tests/trainer/test_sharded_checkpoint.py

```diff
@@ -91,6 +91,8 @@ def _compare_model_params_between_state_dicts(state_dict1, state_dict2):
 @pytest.mark.gpu
 @world_size(2)
 def test_fsdp_full_state_dict_save(world_size, tmp_path: pathlib.Path):
+    if version.parse(torch.__version__) < version.parse('1.13.0'):
+        pytest.skip()
```

Comment:
I think it would be better to use a pytest marker here, for example:
https://github.com/mosaicml/composer/blob/be406894265e6540c2a5d603a6b66d499432ce73/tests/trainer/test_fsdp.py#L19